zwave_js/rootfs/usr/share/tempio/zwave_config.conf

```diff
@@ -14,5 +14,5 @@
         "S2_Authenticated": "{{ .s2_authenticated }}",
         "S2_Unauthenticated": "{{ .s2_unauthenticated }}"
     },
-    "enableSoftReset": {{ .soft_reset }}
+    "enableSoftReset": {{ .soft_reset }}{{ .presets }}
```

Comment:
Why do we concatenate the presets including the key instead of writing the key in the template? We know what they key is in advance, right?

Reply:
we can change this, with the original implementation we didn't know which keys we were going to include hence this hack